src/modules/choose.ts

```diff
@@ -0,0 +1,33 @@
+import IModule from "../module"
+import MessageLike from "../message-like"
+import Ai from "../ai"
+import config from "../config"
+
+export default class DiceModule implements IModule {
+	public readonly priority = 0
+	public readonly name = "choose"
+	public readonly commands = [
+		{
+			name: "choose",
+			desc: "choose one from given N choices"
+		}
+	]
+	private ai: Ai
+
+	public install(ai: Ai) {
+		this.ai = ai
+	}
+
+	public async onCommand(msg: MessageLike, cmd: string[]): Promise<boolean> {
+		if (cmd[0] == "choose") {
+			if (cmd.length < 2) {
+				msg.reply("Usage: /choose <choice 1> <choice 2> ... <choice N>")
+			} else {
+				let choice = Object(cmd).length - 1
```

Comment:
`cmd.length - 1` でも大丈夫そうな気がする
```suggestion
				const choice = cmd.length - 1
```

Reply:
それはよくわからん挙動をするらしいという話を拾ったので敢えてそうしてある（このファイルのやっている範疇では全然問題ないかもしれないけど）

Reply:
多分[これ](https://www.atmarkit.co.jp/ait/articles/1412/15/news119_3.html)を拾ってきたような気がする（もしよければ参考にしたものを教えてもらえると助かる）が、このようなことは発生しないようなコードになっているはずなので多分 suggestion を commit してもらっても大丈夫

Reply:
拾ってきた先はそうです，そう言うならcommitします